# vector of node ids
int32[] node_ids

# vector of ground truth classes
int32[] ground_truth
